rest_api/common/src/paging.rs

```diff
@@ -0,0 +1,168 @@
+// Copyright 2018-2022 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use serde::{Deserialize, Serialize};
+
+pub const DEFAULT_LIMIT: usize = 100;
+pub const DEFAULT_OFFSET: usize = 0;
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq)]
+pub struct Paging {
+    pub current: String,
+    pub offset: usize,
+    pub limit: usize,
+    pub total: usize,
+    pub first: String,
+    pub prev: String,
+    pub next: String,
+    pub last: String,
+}
+
+pub fn get_response_paging_info(
```

Comment:
Feels like this should be PagingBuilder.

Reply:
Added builder